From 936b17e1ba3d19c91a0c24d66a538454100e051e Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Mon, 11 Jun 2018 17:50:25 +0530
Subject: [PATCH 0370/1851] dt-bindings: display: xlnx: mixer: Adding xlnx vtc
 bridge support

commit 5a162b31bf39e029a591291a06c541306c5f61d7 from
https://github.com/Xilinx/linux-xlnx.git

Update documentation showing how to add vtc bridge node to Mixer node.
It is an optional property.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
index 94d43cd00345..0ed4705c7b4e 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
@@ -40,6 +40,8 @@ Optional properties:
  - xlnx,layer-scale: denotes layer can be scale to 2x and 4x
  - xlnx,logo-layer: denotes logo layer is enable
  - logo: logo layer
+ - xlnx,bridge: phandle to bridge node.
+   This handle is required only when VTC is connected as bridge.
 
 Supported Formats:
 	Mixer IP Format		Driver supported Format String
@@ -79,6 +81,7 @@ Example:
 		xlnx,ppc = <2>;
 		xlnx,num-layers = <8>;
 		xlnx,logo-layer;
+		xlnx,bridge = <&v_tc_0>;
 
 		mixer_port: mixer_port@0 {
 			reg = <0>;
-- 
2.31.1

